crates/primitives/src/signature/mod.rs

```diff
@@ -2,9 +2,11 @@ mod error;
 pub use error::SignatureError;
 
 mod parity;
+#[deprecated(since = "0.8.15", note = "see https://github.com/alloy-rs/core/pull/776")]
 pub use parity::Parity;
 
 mod sig;
+#[deprecated(since = "0.8.15", note = "use PrimitiveSignature instead")]
```

Comment:
could we put those on the struct defs instead?

Reply:
Fixed it and added some `#[allow(deprecated)]`